pkg/api/controller.go

```diff
@@ -12,20 +12,27 @@ import (
 	"github.com/anuvu/zot/errors"
 	"github.com/anuvu/zot/pkg/log"
 	"github.com/anuvu/zot/pkg/storage"
+	"github.com/boltdb/bolt"
 	"github.com/gorilla/handlers"
 	"github.com/gorilla/mux"
 )
 
+const (
+	DBFilePath = "./data/db/ZotSearch.db"
+)
+
 type Controller struct {
 	Config     *Config
 	Router     *mux.Router
 	ImageStore *storage.ImageStore
 	Log        log.Logger
 	Server     *http.Server
+	DBPath     string
+	DB         *bolt.DB
```

Comment:
Rather than make people Close() the DB explicitly, can we make a wrapper Close() method? This way if we add more things that need to be closed to this structure, we don't have to go update all the callers again. It also has a nice benifit of preventing an abstraction leak. Unrelated, but does most of this stuff need to be public?

Reply:
Yes, it does not need to be public as I am using in same package. I will also add a wrapper Close() method.